ant-local-testnet-action

```diff
@@ -0,0 +1 @@
+Subproject commit 600912168508b206c51f974440d2316b06618a98
```

Comment:
What is this file needed for?

Reply:
This seemed to be a ci file, but I had a git issue so could not tell, so I left it in place in case it was used